implementations/rust/ockam/ockam_command/src/node/util.rs

```diff
@@ -65,7 +65,17 @@ pub async fn start_embedded_node_with_vault_and_identity(
     let options = TcpListenerOptions::new();
     let listener = tcp.listen(&bind, options).await?;
 
-    let projects = cfg.inner().lookup().projects().collect();
+    let projects = {
```

Comment:
This code seems to be duplicated with [this one](https://github.com/build-trust/ockam/pull/5018/files#diff-afac108b9e4f2377c7932bef242708078b72d74836268ccc0ff4a09a24f67a53R273). Can we refactor it?

Reply:
Done!